src/main/java/passableleaves/core/PLCore.java

```diff
@@ -27,7 +29,7 @@ public PLCore() {
 
     @Override
     public String[] getASMTransformerClass() {
-        return new String[0];
+        return new String[] {"passableleaves.core.PLTransformer"};
```

Comment:
Yep, I tested it before releasing 5.1.0.0 ... to be fair though, I only loaded up a default world in creative and made sure I could walk through leaves... is there something specific I should be looking for?